src/ansi-c/expr2c.cpp

```diff
@@ -41,6 +41,21 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "expr2c.h"
 #include "expr2c_class.h"
 
+namespace {
+  std::string escape(const std::string &str)
+  {
+    std::string result;
+    result.reserve(str.size());
+    for(const auto &ch : str)
+    {
+      if(ch=='"')
```

Comment:
They are not actually escaped, they are just represented as unicode (e.g. `\u0008`), which is fine. However, i see two other issues:
1. Characters `\u000a` and `\u000d` are not authorised at all in a Java String, meaning that the test will not be compilable if these characters are present. Thus, they should be replaced by `\n` and `\r` respectively.
2. The single quote `'` should be escaped in the case of characters (but not in ID_string), and probably also the two above.